README.rst

```diff
@@ -90,6 +76,14 @@ Astropy is licensed under a 3-clause BSD style license - see the
 `LICENSE.rst <LICENSE.rst>`_ file.
 
 
+.. |Astropy Logo| image:: https://github.com/jeffjennings/repo_stats/blob/main/dashboard_template/astropy_banner_light.svg
```

Comment:
I think this PR should be held back from merging until all these linked images here and below can be sourced from an astopy-org-hosted repo that has a bus factor/access >>1 person.

Reply:
Great, that makes sense. It seems the easiest way would be to duplicate the repository to within the astropy organization; following the GitHub [guide](https://docs.github.com/en/repositories/creating-and-managing-repositories/duplicating-a-repository), can someone with the permission to create a new (empty) repo in the organization with the name `repo_stats` do that and let me know? I can then submit a PR there with a mirror of the current repo with updated image hyperlinks. I can also coordinate with that person to add the needed GitHub Actions secrets.

Reply:
If you give me admin access to your repo, I can probably do a straight transfer. We can coordinate privately if you want.

Reply:
Ok yes I've sent you an admin invite, and I'll open a PR in the new repo when it's transferred to update the links.

Reply:
Transfer is complete. Thanks, Jeff! Please refer to https://github.com/astropy/repo_stats now. 🙇‍♀️

Reply:
Great, thank you!